spec/collator.html

```diff
@@ -31,9 +31,7 @@ <h1>InitializeCollator ( _collator_, _locales_, _options_ )</h1>
       <p>
         The abstract operation InitializeCollator accepts the arguments _collator_ (which must be an object), _locales_, and _options_. It initializes _collator_ as a Collator object. The following steps are taken:
       </p>
-      <p>
```

Comment:
why should we delete this?

Reply:
Because it is now covered in the section on language tags, which is automatically linked to by the "language tag nonterminal" phrase in step 11.a.
> This specification identifies locales using _Unicode BCP 47 locale identifiers_ as defined by [Unicode Technical Standard #35 Part 1 Core, Section 3.3 BCP 47 Conformance](https://unicode.org/reports/tr35/#BCP_47_Conformance), and its algorithms refer to **_language tag nonterminals_** defined in the grammars of [Section 3 Unicode Language and Locale Identifiers](https://unicode.org/reports/tr35/#Unicode_Language_and_Locale_Identifiers).